Services/CO.CDP.Organisation.WebApi/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -9,6 +9,7 @@ public static class ServiceCollectionExtensions
     private static readonly Dictionary<Type, (int, string)> ExceptionMap = new()
     {
         { typeof(BadHttpRequestException), (StatusCodes.Status422UnprocessableEntity, "UNPROCESSABLE_ENTITY") },
+        { typeof(MissingOrganisationIdException), (StatusCodes.Status422UnprocessableEntity, "UNPROCESSABLE_ENTITY") },
```

Comment:
It should be 404. I considered 403 too, but we'd be giving out information to the caller this way. So 404.

The code should be something dedicated to this error code, i.e. `ORGANISATION_NOT_FOUND`.

Reply:
done.